app/jobs/phone_number_opt_out_sync_job.rb

```diff
@@ -0,0 +1,42 @@
+class PhoneNumberOptOutSyncJob < ApplicationJob
+  include GoodJob::ActiveJobExtensions::Concurrency
+
+  good_job_control_concurrency_with(
+    total_limit: 1,
+    key: -> do
+      rounded = TimeService.round_time(time: arguments.first, interval: 1.hour)
+      "phone-number-opt-out-sync-#{rounded.to_i}"
+    end,
+  )
+
+  def perform(_now)
+    if !IdentityConfig.store.sms_resubscribe_enabled
```

Comment:
The reason for this feature flag is to simplify coordination of the API permissions infra deploy